MapboxNavigation/NavigationViewLayout.swift

```diff
@@ -23,6 +23,11 @@ extension NavigationView {
         
         wayNameView.centerXAnchor.constraint(equalTo: centerXAnchor).isActive = true
         wayNameView.bottomAnchor.constraint(equalTo: bottomBannerContainerView.topAnchor, constant: -10).isActive = true
+        
+        speedLimitView.topAnchor.constraint(equalTo: topBannerContainerView.bottomAnchor, constant: 10).isActive = true
+        speedLimitView.leadingAnchor.constraint(equalTo: safeLeadingAnchor, constant: 10).isActive = true
```

Comment:
Unlike #1242, this PR places the speed limit indicator in the upper-left corner of the map view. The lower-right corner is already taken up by the ℹ️ button and can also overlap with the current road name label:

<img src="https://user-images.githubusercontent.com/1231218/71517416-d1ccfe00-2862-11ea-9cf5-35fd6dc97975.png" width="300" alt="bridge">